python/ray/data/_internal/execution/resource_manager.py

```diff
@@ -374,12 +366,15 @@ def __init__(self, resource_manager: ResourceManager, reservation_ratio: float):
 
         self._idle_detector = self.IdleDetector()
 
+    def _is_op_eligible(self, op: PhysicalOperator) -> bool:
+        """Whether the op is eligible for memory reservation."""
+        # Only consider operators that have enabled throttling
+        # and hasn't completed.
```

Comment:
Nit: IMO we should remove this comment. Don't think it adds anything since it verbatim describes the code below.
```suggestion
```